src/Symfony/Component/Messenger/Bridge/Amqp/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ CHANGELOG
 
  * Deprecated the `prefetch_count` parameter, it has no effect and will be removed in Symfony 6.0.
  * `AmqpReceiver` implements `QueueReceiverInterface` to fetch messages from a specific set of queues.
+ * Add ability to distinguish retry and delay actions.
```

Comment:
no final dot please (to match our recent policy related to changelogs)

Reply:
Removed final dot. (I was looking at the previous records)